scripts/smoketest.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+import requests
+import time
+
+import pytest
+
+api_url = "http://localhost:8000/general/v0.0.4/general"
+
+def send_document(filename):
+    files = {"files": (str(filename), open(filename, "rb"), "text/plain")}
+    return requests.post(api_url, files=files)
+
+@pytest.mark.parametrize(
+    "example_filename",
+    [
+        "alert.eml",
+        "announcement.eml",
+        "fake-email-attachment.eml",
+        "fake-email-image-embedded.eml",
+        "fake-email.eml",
+        "fake-html.html",
+        pytest.param("fake-power-point.ppt", marks=pytest.mark.xfail(reason="See CORE-796")),
+        "fake-text.txt",
+        "fake.doc",
+        "fake.docx",
+        "family-day.eml",
+        pytest.param("fake-excel.xlsx", marks=pytest.mark.xfail(reason="not supported yet")),
+        # Note(austin) The two inference calls will hang on mac with unsupported hardware error
+        # Need to handle this better
+        "layout-parser-paper.pdf",
+        "layout-parser-paper-fast.jpg",
+    ]
+)
+
+def test_happy_path(example_filename):
+    """
+    For the files in sample-docs, verify that we get a 200
+    and some structured response
+    """
+    time.sleep(1)  # Avoid a 429
```

Comment:
for my own sake: what's the deal with the 429? I'm assuming this has to do with multiple back-to-back calls with the parameterization?

part 2: do we need the sleep at the beginning and end of the test?

Reply:
(Although this pr is getting hairier than I expected so maybe in a new branch)

Reply:
Ahhh, rate limiting, that makes sense now. 

Ohh nice! 🤞

Reply:
ohh yea, totally, and doesn't need to block this pr either if you'd rather cut after you get this in?